docs/Cask-Cookbook.md

```diff
@@ -303,7 +303,7 @@ The [`find-appcast`](https://github.com/Homebrew/homebrew-cask/blob/HEAD/develop
 
 | key             | value       |
 | --------------- | ----------- |
-| `must_contain:` | a custom string for `brew audit --appcast {{cask_file}}` to check against. |
+| `must_contain:` | a custom string for `brew audit --appcast <cask_file>` to check against. |
```

Comment:
Should this just be
```suggestion
| `must_contain:` | a custom string for `brew audit --appcast <cask>` to check against. |
```

Or
```suggestion
| `must_contain:` | a custom string for `brew audit --appcast <token>` to check against. |
```

Reply:
I guess we do use `<cask>` in `livecheck` docs:
https://github.com/Homebrew/brew/blob/master/docs/Brew-Livecheck.md#creating-a-check
```
brew livecheck --debug <formula>|<cask>
```